examples/runqslower/src/bpf/runqslower.bpf.c

```diff
@@ -1,9 +1,15 @@
 // SPDX-License-Identifier: GPL-2.0
 // Copyright (c) 2019 Facebook
+
 #include "vmlinux.h"
 #include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
 #include "runqslower.h"
 
+struct task_struct_pre_5_14 {
```

Comment:
this won't work, you need to use three underscored after `task_struct` to make libbpf recognize it as just another flavor of task_struct